appendix-11-asking for help.md

```diff
@@ -0,0 +1,48 @@
+
+### How long do you need?
+1 minute, 15 minutes, or an hour help them understand what their committing to?
+
+### What is this about?
+Please help me understand what I'll be contributing to. Is this design, code, product UX, your development env?
+
+### What sort of attention do you need from me?
+Do you need me to listen, explain, or help you fix something? Aka, what do I need to prepare to give.
+
+Each of the above things allows a requestee to determine what helping you is going to require of them.
+
+## Requestee, please communicate your situation
+As much as it's essential for the person asking to communicate their needs, it's equally vital that you help them understand your situation. If the requester has done their job of "communicating their need," which is not always the case, success now depends on you responding in kind by communicating your situation or maybe even asking them to express their need more clearly.
+
+That said, just because someone has asked for your time doesn't mean you're in a ready state and position to give it. The expectation is that you communicate your situation by:
+
+### Ask clarifying questions
+Needs are not always clearly communicated. It's now your job to help them understand what you don't. Maybe this means asking for more info about the problem or asking clarifying questions like, "do you need to show me something, or can this just be a phone call?".
+
+### Communicating your obstacles
+As we said, you might not be free, and it's ok to communicate that. "I'm in the zone and won't be out available 1pm", or "I've got my child with me for the next hour" or even "I can talk, but my kids are in the room so it might be a bit loud."
+
+Doing these things allows the requestor to understand your situation and decide how to proceed.
+
+While these may seem simple, and they can be, they are often quickly forgotten when asking someone for help. Practicing our ability to communicating our needs and situation will make us a more reliable remote team, who respect our teammates and optimize our time spent collaborating.
+
+## Be understanding
+But what if the person I'm asking for help doesn't respond? This is a real possibility; maybe they are away from their phone or just in the zone. Their lack of response might feel like they aren't communicating. However, if we step out of our own selfish need for a second, we'll see that they are communicating. Their silence is telling us that they're not available. We need to be able to respect that.
+
+Does the practice "being understanding" give a coworker carte blanche to ignore your request for help for weeks, no. It is, however, essential for us to understand that a primary goal of being a remote team is to create space for everyone to focus. So if they don't respond, they're probably just busy.
```

Comment:
> Does the practice "being understanding"

'Does the practice of "being understanding"'

> ignore your request for help for weeks, no.

"ignore your request for help for weeks? No."